bin/install-checkstyle.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+curl -s -i https://sourceforge.net/projects/checkstyle/files/latest/download | \
```

Comment:
Do we really want to just install the latest? I think there's value in being explicit about which version we're using, so that it's easy to determine by reading the code, and so we know exactly what to change to perform an upgrade.

Reply:
I wanted an automated way to upgrade, but maybe this is not the pattern we are following in the other engines and I see your point.
What about turning this into a 2-step thing? Instead of calling this directly from `Dockerfile` we would `make upgrade` first, then `make image` would have the latest... 
...hmm, then we would have to commit those files, not sure I like this idea anymore.
Probably simpler just being explicit.

Reply:
Ah! I could have a `make` target just to show the latest URL. Then use it explicitly.